maps/away/ships/coc/coc_ranger/coc_ship.dm

```diff
@@ -2,7 +2,7 @@
 	name = "Ranger Gunboat"
 	description = "The Xansan-class is not, in fact, a distinct design in of itself. It is instead Xanu Prime’s variant of the Lagos-class gunboat, a Solarian light attack ship design. While the Lagos-class has been out of service with the Alliance’s navy for centuries, the blueprints were captured during the Interstellar war by Xanu Prime militiamen from a regional naval shipyard during the opening stages of the conflict, and were immediately used to construct ships of the class for use by Xanu Prime’s burgeoning military. While the Xansan-class has been retired from Xanu service, the leftover ships were donated en masse to the Frontier Marshal Bureau, and the Rangers make use of the craft to this day, in spite of their advanced age."
 	suffixes = list("ships/coc/coc_ranger/coc_ship.dmm")
-	sectors = list(SECTOR_BADLANDS, ALL_COALITION_SECTORS)
+	sectors = list(SECTOR_BADLANDS, SECTOR_WEEPING_STARS, SECTOR_COALITION, SECTOR_LIBERTYS_CRADLE, SECTOR_ARUSHA)
```

Comment:
Do this via the sectors_blacklist var added in <https://github.com/Aurorastation/Aurora.3/pull/17733>